### PR TITLE
Maxlevel

### PR DESCRIPTION
**Pull request by daedalus**
small code improvement.

### CODE REVIEW

File: FACTOR.py [hunk 1]
```diff
@@ -39,6 +37,15 @@
 RPC_PASS     = os.environ.get("RPC_PASS", "rpcpass") 
 SCRIPTPUBKEY = os.environ.get("SCRIPTPUBKEY") 
 
+SIEVE_MAX_LEVEL = os.environ.get("SIEVE_MAX_LEVEL")
+if SIEVE_MAX_LEVEL == None: 
+  SIEVE_MAX_LEVEL = 31
```
[the remark this answers is not in the document]
Comment (daedalus) [reply FACTOR.py:42]: That is the current value...

[the remark this answers is not in the document]
Comment (daedalus) [reply FACTOR.py:42]: do you want me to change it?

[the remark this answers is not in the document]
Comment (ReverseControl) [reply FACTOR.py:42]: It should be up to 30, including 30. But, not 31. I misspoke, I was looking at the size of the numbers stored in hex as strings, so the actual size is half of I quoted. So about 175MB for level 30 and 365 for level 31.   Is https://github.com/FACT0RN/factoring/blob/main/FACTOR.py#L30 inclusive on the ending part of the range.

[the remark this answers is not in the document]
Comment (daedalus) [reply FACTOR.py]: Please take a look a the recent changes

**Pull request merged**